doc/ref_model/chapters/chapter08.md

```diff
@@ -1,4 +1,320 @@
+</tr>
+<tr><td>infra.stg.cfg.003</td><td>Storage with replication</td><td></td><td>N</td><td>Y</td><td>N</td><td>Optional</td></tr>
+<tr><<td>infra.stg.cfg.004</td><td>Storage with encryption</td><td></td><td>Y</td><td>Y</td><td>N</td><td>Optional</td></tr>
+<tr><<td>infra.hw.cpu.cfg.001</td><td>Minimum Number of CPU sockets</td><td>This determines the minimum number of CPU sockets within each host</td><td>2</td><td>2</td><td>1</td><td>1</td></tr>
+<tr><<td>infra.hw.cpu.cfg.002</td><td>Minimum Number of cores per CPU</td><td>This determines the number of cores needed per CPU.</td><td>20</td><td>20</td><td>2</td><td>2</td></tr>
+<tr><<td>infra.hw.cpu.cfg.003</td><td>NUMA alignment</td><td>NUMA alignment support and BIOS configured to enable NUMA</td><td>N</td><td>Y</td><td>N</td><td>Optional</td></tr>
+</table>
+
+<a name="8.3.4"></a>
+### 8.3.4  Telco Edge Cloud Infrastructure Characteristics
+This section characterises the hardware capabilities for different edge deployments and the Platform services that run on the infrastructure. Please note, that the Platform services are containerised to save resources, and benefit from intrinsic availability and auto-scaling capabilities.
+
+**Table 8-4. Characteristics of Infrastructure nodes
+
+<table>
+<tr>
+<th></th>
+<th colspan="7">Platform Services</th>
+<th colspan="3">Storage</th>
+<th colspan="3">Network Services</th>
+</tr>
+<tr>
+<th></th>
+<th>Idnetity</th>
+<th>Image</th>
+<th>Placement</th>
+<th>Compute</th>
+<th>Networking</th>
+<th>Message Queue</th>
+<th>DB Server</th>
+<th>Ephemeral</th>
+<th>Persistent Block</th>
+<th>Persistent Object</th>
+<th>Management</th>
+<th>Underlay (Provider)</th>
+<th>Overlay</th>
+</tr>
+<tr>
+<td>Control Nodes</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td></td>
+<td>&#9989;</td>
+<td></td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+</tr>
+<tr>
+<td>Workload Nodes<br>(Compute)</td>
+<td></td>
+<td></td>
+<td></td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td></td>
+<td></td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+</tr>
+<tr>
+<td>Storage Nodes</td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+<td></td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+<td>&#9989;</td>
+</tr>
+</table>
+
+Depending on the facility capabilities, deployments at the edge may be similar to one of the following:
+        - Small footprint edge device
+        - Single server: deploy multiple (one or more) workloads
+        - Single server: single Controller and multiple (one or more) workloads
+        - HA at edge (at least 2 edge servers): Multiple Controller and multiple workloads
+
+
+<a name="8.3.5"></a>
+### 8.3.5 Comparison of Edge terms from various Open Source Efforts
+
+**Table 8-5. Comparison of Edge Terms
+
+<table>
```

Comment:
Please do not use raw html tables.

Reply:
@CsatariGergely This table needed to be in HTML because column spans

Reply:
@CsatariGergely replaced html with md - formatting not as good